skins/laika/src/components/shared/Email.vue

```diff
@@ -25,6 +26,7 @@ export default Vue.extend( {
 	},
 	props: {
 		initialValue: String,
+		showError: Object as () => AddressValidity,
```

Comment:
This should be a boolean field. Better do `fieldErrors.email` in the Address component.